examples/advanced/llm_hf/requirements.txt

```diff
@@ -0,0 +1,8 @@
+nvflare
+torch
+datasets
+tensorboard
+transformers
```

Comment:
Do we have license for this "transformers"?

Reply:
license added